fix: Improve background running notification

kitadai31 opened the pull request:
Several improvements for the notification which appears while patcher is running.

Changes:

### 1. Remove large icon 

The large icon was unnecessary.
Reason:

- According to the developers document, this icon must not be the app icon.
https://developer.android.com/develop/ui/views/notifications#Templates
- It didn't look good in light mode.
See https://github.com/ReVanced/revanced-manager/issues/2567#issuecomment-2999108092 for a image.

### 2. Use importance `IMPORTANCE_LOW` instead of `IMPORTANCE_HIGH`

`IMPORTANCE_HIGH` = Status bar icon + Sound + Heads-up banner.
`IMPORTANCE_LOW` = Status bar icon only.

Semantically, this notification is not important as it is merely necessary to run the foreground service.
So the latter is suitable for it.
This resolves the annoyance of a notification sound and Heads-up banner each time you start applying a patch.

(`IMPORTANCE_MIN` cannot be used for foreground services.)

### 3. ~~Remove unused intent~~ Fix intent

Currently, the notification responds to a tap but nothing happens.
Fixed the intent to resolve this issue.

### 4. Set category

Following best practices, set a notification category.

### 5. Improve text

Adjusted notification title and text.
Using the app title to notification title is not recommended.

Also, add a message "Tap to return to the patcher" like flutter manager.

## Screenshots

Before PR:
![before](https://github.com/user-attachments/assets/061fda4e-e137-46a7-8f2e-b3e65863f35c)

After PR:
![after](https://github.com/user-attachments/assets/9052ad75-814a-479b-bc5e-8eb130dfafc9)

For reference, the Flutter Manager's notification:
![flutter](https://github.com/user-attachments/assets/195d985e-6194-4140-bf4f-ee4b08009186)

brosssh commented:
Does the notification get automatically dismissed once the patching process is over? 
Ideally the notification should stay there until the process is done, and if user want to get back to the patching screen they can simply press the notification.

brosssh commented:
> Should I restore and fix the notification intent?

I would say yes, thank you.

Axelen123 commented:
I agree that tapping should bring the user back to the patcher screen.

kitadai31 commented:
Done.

I updated the PR description based on the latest commit.
Added the screenshots.
Please take a look.

Now I'm struggling with PR title.
Please update the title as needed.
